packages/core/tests/test_simple_for_testing.py

```diff
@@ -5,13 +5,14 @@
 workflow, logic etc.
 """
 from dagster import build_op_context
-from bag3d.core.simple_for_testing import test_table1, test_table2
+#from bag3d.core.simple_for_testing import test_table1, test_table2
 
 
-def test_tables(resource_db_connection_docker, resource_container):
+def test_tables(database):
     context = build_op_context(
-        resources={"container": resource_container,
-                   "db_connection": resource_db_connection_docker}
+        resources={"db_connection": database}
     )
-    res1 = test_table1(context)
-    res2 = test_table2(context, res1)
+    # TODO: FIX THIS
```

Comment:
As discusssesd, these were probably transferred from core to common but also they are not very important - so I temporarily disabled the test to remember to fix it!